.github/config/profile.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 profile: PLACEHOLDER_PROFILE
-email: aiida@localhost
```

Comment:
Not crucial, but since we decided to keep this as allowed, maybe revert it

Reply:
Done!